lib/filter.rb

```diff
@@ -49,9 +49,9 @@ class Filter
   def self.parse_filter_definitions(filter_definitions)
     element_filters = {}
     Array(filter_definitions).each do |definition|
-      path, matcher_definition = definition.split("=", 2)
+      path, operator, matcher_definition = definition.scan(/(.*?)(=|!=)(.*)/)[0]
```

Comment:
A filter definition like `/path>=matcher` will silently have no effect as a path `/path>` can't exist in the data. Maybe it would be good to catch these kind of cases, so that users trying different operators get proper feedback.